frontend/src/scenes/session-recordings/playlist/playlistUtils.ts

```diff
@@ -108,25 +106,12 @@ export async function createPlaylist(
     playlist: Partial<SessionRecordingPlaylistType>,
     redirect = false
 ): Promise<SessionRecordingPlaylistType | null> {
-    try {
-        playlist.filters = playlist.filters || DEFAULT_RECORDING_FILTERS
-        const res = await api.recordings.createPlaylist(playlist)
-        if (redirect) {
-            router.actions.push(urls.replayPlaylist(res.short_id))
-        }
-        return res
-    } catch (e: any) {
-        if (e.status === 403) {
-            openBillingPopupModal({
-                title: `Upgrade now to unlock unlimited playlists`,
-                description: PLAYLIST_LIMIT_REACHED_MESSAGE,
-            })
-        } else {
-            throw e
-        }
+    playlist.filters = playlist.filters || DEFAULT_RECORDING_FILTERS
```

Comment:
playlist count is never gated anymore so we didn't need the billing modal fallback here